services/manage/app/templates/components/team/modal-form.hbs

```diff
@@ -79,7 +79,8 @@
       </div>
     {{/modal.body}}
 
-    {{#modal.footer}}
+    {{#modal.footer class="justify-content-between"}}
+      <FormElements::ToggleButton @value={{model.active}} @onLabel="Active" />
```

Comment:
While technically it doesn't matter, the new Ember docs want to use `this` when referencing a value in the containing component... as such `@value={{this.model.active}}` - i'm trying hard to switch to this style (though I forget at times)